NIPostsReload/Presentation/Scenes/NIPostFeedModule/Presenter/NIPostFeedPresenter.swift

```diff
@@ -81,6 +102,48 @@ final class DefaultNIPostFeedPresenter: NIPostFeedPresenter {
         
         view.updateCollectionViewLayout()
     }
+    
+    func searchPosts(query: String?) {
+        searchWorkItem?.cancel()
+        
+        guard let query, !query.isEmpty else {
+            updatePosts()
+            return
+        }
+        
+        guard query.count >= 2 else {
```

Comment:
Why does the search initiate only after two elements are entered? This might confuse users who don't understand why entering a single letter doesn't trigger any action